### ISSUE TITLE
Retrieve request for https://unicode.org/emoji/charts/full-emoji-list.html creates unretrievable message in rabbitmq

### ISSUE THREAD

**Issue by webignition** [webignition/async-http-retriever#308]
**Describe the issue**

A request for https://unicode.org/emoji/charts/full-emoji-list.html results in an irretrievable (unacked) message in rabbit-mq.

Rabbit-mq shows a single unacked message in the queue. This message cannot be purged. The management interface states that there are no messages when attempting to retrieve one message from the queue.

**To Reproduce**

Stop the `app-cli` service. Add a `MESSENGER_DSN` env var to `app-web`. This lets us selectively consume messages.

Request https://unicode.org/emoji/charts/full-emoji-list.html.

```bash
curl -X POST http://localhost:8001/ \
     -d "url=https://unicode.org/emoji/charts/full-emoji-list.html&callback=http://callback.example.com"
```

This results in a single acked message in rabbit-mq. This message can be retrieved and can be seen to correctly be the relevant serialized RetrieveResource message.

```bash
docker-compose exec app-web /bin/bash
export MESSENGER_TRANSPORT_DSN=amqp://guest:guest@rabbitmq:5672/%2f/messages
./bin/console messenger:consume-messages
```
After a short while, the rabbit-mq management interface reports a single unacked message which cannot be retrieved from the queue.

~The queue contents cannot be examined, however it seems to be that the RetrieveResource message was processed, something happened and a SendResponse message, one that cannot be acked, has been placed into the queue.~

RetrieveResourceHandler gets stuck retrieving the resource ...

**Comment by webignition** [comment]
HTML for that URL is about 43 megabytes. It just takes a while.

Need, however, to look into how `messenger:consume-messages` can run parallel workers.

**Comment by webignition** [comment]
Not an issue, but the fact that the consumption/handling of the current message delays consumption/handling of queued messages will be addressed in #311.
